eclass/xdg.eclass

```diff
@@ -35,7 +35,7 @@ fi
 xdg_src_prepare() {
 	xdg_environment_reset
 
-	has ${EAPI:-0} 6 && default
+	[[ ${EAPI:-0} == ]] || default
```

Comment:
I guess you are missing something there. ;) I'd rather make it `!= [45]` though.

Reply:
Ackh. It *was* right but after retooling it I missed that. Will fix. Hmm. Yeah, != [invalideapis]
does have a bit of a better flow to it.